src/test/regress/sql/multi_generate_ddl_commands.sql

```diff
@@ -108,14 +108,22 @@ CREATE FOREIGN TABLE foreign_table (
 
 SELECT master_get_table_ddl_events('foreign_table');
 
+SELECT create_distributed_table('foreign_table', 'id');
+ALTER FOREIGN TABLE foreign_table rename to renamed_foreign_table;
+ALTER FOREIGN TABLE renamed_foreign_table rename full_name to rename_name;
+ALTER FOREIGN TABLE renamed_foreign_table alter rename_name type char(8);
+\c - - - :worker_1_port
+\d renamed_foreign_table_*
```

Comment:
We (now) try to avoid \d in regression tests because its output tends to change heavily across major releases.

Reply:
I've updated the test to select from `information_schema.columns`